assignments/cyb5-service-architecture.md

```
@@ -1,39 +1,1398 @@
+        library.save();
+    }
+}
+```
+
+### Command Reference
+
+Your CLI must support the following commands. Each command has a required syntax, behavior, and output format. Where output format is specified, your CLI must match it closely enough for automated testing (exact whitespace is not tested, but structure and content are).
+
+#### `help` — Contextual Help
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)